youtube_dl/extractor/rai.py

```diff
@@ -208,10 +208,31 @@ def _real_extract(self, url):
         }
 
         info.update(relinker_info)
-
         return info
 
 
+class RaiPlayLiveIE(RaiBaseIE):
+    _VALID_URL = r'(?P<url>https?://(?:www\.)?raiplay\.it/dirette/(?P<id>\w*))'
+    _TEST = {
+        'url': 'http://www.raiplay.it/dirette/rai3',
+        'only_matching': True,
+    }
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        url, channel = mobj.group('url', 'id')
```

Comment:
What's the point capturing `url` when you already have it in `url`. Rewrite with `_match_id`.